### ISSUE TITLE
Customizing Modal Dialog size and Highlighted Step

### ISSUE THREAD

**Issue by dualistique39** [okgrow/react-native-copilot#69]
Copilot has been working great off the bat, but I need some guidance on how I am able to customize the size of the modal window that's being generated and the highlighted step.  When there's a spot highlighted, there seems to be a few pixels added making the highlight not exact to my tabs.

Also, is there some options or some way I can determine the size of the modal.  This comes an issue when I have a middle tab button highlighted, and my modal window is much smaller size it draws the modal to the right.  

![test](https://user-images.githubusercontent.com/35152678/44886402-cff92780-ac7a-11e8-9b8b-cd8882192358.png)

**Comment by dualistique39** [comment]
@mohebifar - Thanks for your comment and pointing out the algorithm.  How about the highlighted "tab/icon" itself.  You can see that it's off by a few pixels (from the results that I expect).

Would modifying [this](https://github.com/okgrow/react-native-copilot/blob/55e0db49e6719f7a62bb046202fc449652c42522/src/components/style.js#L8) solve that problem?

**Comment by mohebifar** [comment]
Yes, looking at the screenshots you shared, it seems to be that `OFFSET` thing. If you'd like to make that customizable, PRs are welcome!

**Comment by dnlsandiego** [comment]
@mohebifar is it possible to customize the highlighted window? Like adding a border?

**Comment by oliemansm** [comment]
@mohebifar Running into the same issues. Any movement on this?

**Comment by ViniSchonardie** [comment]
Any updates on this topic?

**Comment by pavandugariya** [comment]
How to use different screen because  when i use android app is lacked
